escavenge.py

```diff
@@ -10,6 +10,9 @@ def __init__(self):
 
```

Comment:
Couldn't add a comment up above so I'm adding it here. database.cpythong-36.pyc, gamemaker.cpython-36.pyc, and team.cpython-36.pyc need to be removed

Reply:
Yup. I can't delete binary files in a PR so we'll have to remove them once the PR is pushed through.